notebooks/community/ml_ops/stage7/model_monitoring_custom.ipynb

```diff
@@ -0,0 +1,1267 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        time.sleep(120)
</pre><p>Can we use <code>monitoring_job.result()</code> rather than <code>sleep()</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>We are waiting for the email. Can play around and see if there is some state change when the email is sent.</p>

Reply:
<p>There is no result() method. I tried using "state" property, but it is not an indicator of a change in state of receiving the email.</p>